compiler/modules/front.syntax/tests/data/testParser.json

```diff
@@ -220,10 +220,10 @@
     ["E", "expr", "[,1]", ""],
 
     { "comment" : "mapping constructor" },
-    ["V", "expr", "{ }"],
-    ["V", "expr", "{x: 1, y: 2}", "{ x: 1, y: 2 }"],
-    ["V", "expr", "{x: 1, y: 2, \"z\": 3}", "{ x: 1, y: 2, \"z\": 3 }"],
-    ["V", "expr", "{value: 1+2}", "{ value: 1 + 2 }"],
+    ["V", "expr", "{ }", "{\n}"],
```

Comment:
We should be able to avoid this: have a flag saying don't add newlines after braces. Set this flag for a mapping constructor.

Reply:
fixed